app/controllers/questions/at_capacity_controller.rb

```diff
@@ -8,6 +8,8 @@ def self.show?(intake)
     end
 
     def edit
+      redirect_to next_path unless current_client.routing_method_at_capacity?
```

Comment:
@alaroia-cfa Even though the .show? method says "don't show unless its at capacity", it seems that the show method actually only gets used in ControllerNavigation. Which means that when there's a direct link to the page, the show logic never runs. 

This is how it has always been BUT I think I didn't really internalize that in the original implemention / maybe wasn't completely aware that that's how it worked.